sla_per_user_system_test.py

```diff
@@ -493,23 +507,71 @@ def test_read_throughput_vs_latency_disk_only(self):  # pylint: disable=invalid-
 
         read_cmds = {'troughput': self.define_read_cassandra_stress_command(user=read_users[0],
                                                                             load_type=self.DISK_ONLY_LOAD,
-                                                                            workload_type=self.WORKLOAD_THROUGHPUT, threads=200,
+                                                                            workload_type=self.WORKLOAD_THROUGHPUT,
+                                                                            threads=200,
                                                                             stress_duration_min=stress_duration,
                                                                             max_rows_for_read=max_key_for_cache*2),
                      'latency': self.define_read_cassandra_stress_command(user=read_users[1],
                                                                           load_type=self.DISK_ONLY_LOAD,
-                                                                          workload_type=self.WORKLOAD_LATENCY, threads=250,
+                                                                          workload_type=self.WORKLOAD_LATENCY,
+                                                                          threads=250,
                                                                           stress_duration_min=stress_duration,
                                                                           max_rows_for_read=max_key_for_cache*3),
                      'latency_only': self.define_read_cassandra_stress_command(user=read_users[1],
                                                                                load_type=self.DISK_ONLY_LOAD,
-                                                                               workload_type=self.WORKLOAD_LATENCY, threads=250,
+                                                                               workload_type=self.WORKLOAD_LATENCY,
+                                                                               threads=250,
                                                                                stress_duration_min=stress_duration,
                                                                                max_rows_for_read=max_key_for_cache)
                      }
 
         self._throughput_latency_tests_run(read_users=read_users, read_cmds=read_cmds, latency_user=read_users[1])
 
+    def test_read_50perc_write_50perc_load(self):
```

Comment:
is there a new pipeline calling this new function ?

Reply:
not yet

Reply:
added